doc/source/guide/glossary.rst

```diff
@@ -74,6 +74,17 @@ Glossary
         the result. Usually less efficient than :term:`in-place evaluation`
         since new memory is allocated and data needs to be copied.
 
+    proximal
+        Given a closed proper convex function :math:`f`, the proximal operator is defined by
+        
+            .. math::
+                
+                \text{prox}_f(v) = \arg\min_x f(x) + (1/2)||x - v||_2^2
+        
+    proximal factory
+        A proximal factory associated with a function :math:`f` is a `callable`, which when called with a scalar :math:`\sigma`
+        returns the proximal of the scaled function :math:`\sigma f`.
```

Comment:
Good.

Perhaps take the "when called with a scalar sigma" part to the end of the sentence, reads better.
A quick note like "This is due to the fact that optimization methods often use `\text{prox}_{\sigma f}` for varying sigma." would clarify things even more.